oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Configuration.java

```diff
@@ -260,6 +260,27 @@
                     "are separated with '::'. Example: -Doak.documentstore.persistentCacheIncludes=/content::/var")
     String[] persistentCacheIncludes() default {"/"};
 
+    @AttributeDefinition(
+            name = "Full GC Include Paths",
+            description = "Paths which should be included in full garbage collection." +
+                    "Empty value means all paths are included. " +
```

Comment:
I find this potentially confusing.

Currently when no include path is specified, it results in "include everything" - so specifying an "include everything" explicitly wouldn't be necessary. But if we wanted to support this, then I think it shouldn't be an empty string, but rather eg `"/"` as that is more explicit (so if someone would specify an empty string, we could rather log a warn and skip that entry).

wdyt?

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/1551/commits/55f310e2d65cfe3208ff7688ce93787eccb0cfba